libsdhcdrivers/src/sdhc.c

```diff
@@ -727,13 +717,13 @@ static int sdhc_reset(sdio_host_dev_t *sdio)
     /* Check if a SD card is inserted. */
     val = readl(host->base + PRES_STATE);
     if (val & SDHC_PRES_STATE_CINST) {
-        printf("Card Inserted");
+        ZF_LOGD("Card Inserted");
         if (!(val & SDHC_PRES_STATE_WPSPL)) {
-            printf("(Read Only)");
+            ZF_LOGD("(Read Only)");
         }
-        printf("...\n");
+        ZF_LOGD("...");
```

Comment:
we can drop this. It worked with printf() because the lines above were not printing a "\n". But with ZF_LOGD() each message will separate now, so this shows up alone on a separate line. So I'd say we just remove this line.

Reply:
Removed on the with the most recent commits.